### ISSUE TITLE
[CT-938] compatibility error dbt-core 1.2.0 and bigquery plugin 1.1.1

### ISSUE THREAD

**Issue by smithj51** [dbt-labs/dbt-core#5559]
Getting a non-fatal compatibility error with the new dbt-core release and latest bigquery plug-in:

![image](https://user-images.githubusercontent.com/102249179/181044966-2068db12-45f5-45fa-b57a-fcc9e048b3cb.png)

**Comment by leahwicz** [comment]
@smithj51 you have to have the same minor version of your dbt adapter (in this case `dbt-bigquery`) installed as `dbt-core`. If you upgrade `dbt-bigquery` to 1.2.0, then this should resolve

**Comment by leahwicz** [comment]
I should also add that this is the command that should do the trick:
`pip install --upgrade  dbt-bigquery`

If it doesn't work, please let me know

**Comment by max-sixty** [comment]
FWIW I've been getting some odd errors around this from poetry:


```
❯ poetry add dbt-bigquery@latest
Using version ^1.2.0 for dbt-bigquery

Updating dependencies
Resolving dependencies... (0.3s)

  SolverProblemError

  Because no versions of dbt-bigquery match >1.2.0,<2.0.0
   and dbt-bigquery (1.2.0) depends on dbt-core (>=1.2.0,<1.3.0), dbt-bigquery (>=1.2.0,<2.0.0) requires dbt-core (>=1.2.0,<1.3.0).
  So, because no versions of dbt-core match >=1.2.0,<1.3.0
   and [proj] depends on dbt-bigquery (^1.2.0), version solving failed.

  at /opt/homebrew/Cellar/poetry/1.1.14/libexec/lib/python3.10/site-packages/poetry/puzzle/solver.py:241 in _solve
      237│             packages = result.packages
      238│         except OverrideNeeded as e:
      239│             return self.solve_in_compatibility_mode(e.overrides, use_latest=use_latest)
      240│         except SolveFailure as e:
    → 241│             raise SolverProblemError(e)
      242│
      243│         results = dict(
      244│             depth_first_search(
      245│                 PackageNode(self._package, packages), aggregate_package_nodes

```

I'm not sure this is necessarily a dbt problem though. 
` pip install --upgrade dbt-bigquery` works for me.

**Comment by smithj51** [comment]
thank you! yes I have a prescript that installs and upgrades all requirements (dbt-core, dbt-query, etc.) and it is working today. It seems there was just a delay in the availability of dbt-bigquery v1.2.0

**Comment by leahwicz** [comment]
@max-sixty are you still seeing that issue?

When we release a new minor version, we do have a time gap between when `dbt-core` is released and then each of the adapters are released. We run re-run test to ensure that the adapter version going out is compatible with the `dbt-core` version that has been released to Pypi. In this instance, there was about an hour between when `dbt-core` v1.2 was released and `dbt-bigquery` v1.2 was released.

**Comment by max-sixty** [comment]
Great, I think it was just the delay — all working for me now!
